core/core-test-toolkit/src/main/java/com/sflpro/notifier/services/notification/impl/DummySimpleEmailSender.java

```diff
@@ -0,0 +1,21 @@
+package com.sflpro.notifier.services.notification.impl;
```

Comment:
Please move this to a package which has `test` in its path.

Reply:
Pushin new changes, pls check,